### PR TITLE
Adding firmware for 2021 Honda CRV Touring

### PR DESCRIPTION
**Pull request by jhlee111**

### CODE REVIEW

File: selfdrive/car/honda/values.py [hunk 1]
```diff
@@ -625,6 +625,7 @@ class CAR:
   },
   CAR.CRV_5G: {
     (Ecu.programmedFuelInjection, 0x18da10f1, None): [
+      b'37805-5PA-AH20\x00\x00', #2021 CRV Touring
```
Comment (VirtuallyChris) [inline selfdrive/car/honda/values.py:628]: Revert the comment

**Comment by VirtuallyChris** [comment]
What's your Dongle ID?

**Comment by jhlee111** [comment]
c73b9fb79dc5a9a5

File: selfdrive/car/honda/values.py (hunk 1 above)
Comment (pd0wm) [inline selfdrive/car/honda/values.py:628]:
```suggestion
      b'37805-5PA-AH20\x00\x00',
```

**Pull request merged**